src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -293,7 +294,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
                 { ShortcutAction::SwitchToTab, RS_(L"SwitchToTabCommandKey") },
                 { ShortcutAction::TabSearch, RS_(L"TabSearchCommandKey") },
                 { ShortcutAction::ToggleAlwaysOnTop, RS_(L"ToggleAlwaysOnTopCommandKey") },
-                { ShortcutAction::ToggleCommandPalette, RS_(L"ToggleCommandPaletteCommandKey") },
+                { ShortcutAction::ToggleCommandPalette, L"" },
```

Comment:
I could be wrong, but I think we actually want this to stay the same? Looking at `MoveFocus` and that needs an arg, but is still defined here.

Reply:
I think there is a lack of consistency, since `MoveFocus` is defined but `SetColorScheme` is not. I am not sure this is important as the code path is not reachable, but probably, if there is a bug in GenerateName I would prefer the command not to appear at all rather than go with the default name.

Reply:
Agreed.